### PR TITLE
add Docker upgrade instructions

### PR DESCRIPTION
**Pull request by eiddor**
Add upgrade instructions for Docker version per #59

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,19 +93,50 @@ $ brew install locast2tuner
...
 $ curl -o $HOME/.locast2tuner/config https://raw.githubusercontent.com/wouterdebie/locast2tuner/main/assets/config.example
-# ... edit the file ...
+# ... edit the file with your settings ...
+# Pull latest locast2tuner image:
 $ docker pull ghcr.io/wouterdebie/locast2tuner:latest
 $ docker run -p 6077:6077 -v $HOME/.locast2tuner/:/app/config --name locast2tuner -d ghcr.io/wouterdebie/locast2tuner:latest
 ```
 
-If you'd like to use `docker-compose` you can use the sample [docker-compose.yml](https://github.com/wouterdebie/locast2tuner/blob/main/assets/docker/docker-compose.yml).
+If you'd like to use `docker-compose` (_recommended_) you can use the sample [docker-compose.yml](https://github.com/wouterdebie/locast2tuner/blob/main/assets/docker/docker-compose.yml) and edit it to match your parameters.
```
Comment (wouterdebie) [inline README.md:107]: Maybe move the recommended way to the top of the section? And then add something like "If you want to do this manually.. bla bla.."?

Comment (wouterdebie) [reply README.md:107]: Or maybe even split them completely?

like:

- Docker
  - docker-compose (recommended)
    - run
    - upgrade 
  - manually
    - run
    - upgrade

Comment (eiddor) [reply README.md:107]: Yeah, I like your second idea - I played with some other formatting ways but was never happy.  I'll work on this in a bit.

Comment (eiddor) [reply README.md]: @wouterdebie Let me know what you think of this format.

**Comment by wouterdebie** [comment]
@eiddor any update on this? Or can I merge this?

**Comment by eiddor** [comment]
If you're happy with it, I think you can merge it!

**Review by wouterdebie — APPROVED** [review]

**Pull request merged**